scripts/cron.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import print_function
+import datetime
+import os
+import sys
+
+from django.core.management import call_command
+from django.conf import settings
+
+import requests
+from apscheduler.schedulers.blocking import BlockingScheduler
+
+
+schedule = BlockingScheduler()
+
+
+class scheduled_job(object):
+    """Decorator for scheduled jobs. Takes same args as apscheduler.schedule_job."""
+    def __init__(self, *args, **kwargs):
+        self.args = args
+        self.kwargs = kwargs
+
+    def __call__(self, fn):
+        job_name = fn.__name__
+        self.name = job_name
+        self.callback = fn
+        schedule.add_job(self.run, id=job_name, *self.args, **self.kwargs)
+        self.log('Registered.')
+        return self.run
+
+    def run(self):
+        self.log('starting')
+        try:
+            self.callback()
+        except Exception as e:
+            self.log('CRASHED: {}'.format(e))
+            raise
+        else:
+            self.log('finished successfully')
+
+    def log(self, message):
+        msg = '[{}] Clock job {}@{}: {}'.format(
```

Comment:
Indeed but it's convenient when you debug locally. I'll find a better way to do this in the future.